tests/CMakeLists.txt

```diff
@@ -195,6 +195,8 @@ if (LIBXML2_FOUND)
         include_directories(${LIBXML2_INCLUDE_DIR})
 	include_directories(externals/Catch/include ${COMMON_INCLUDES})
 	add_executable(test-asebahttp test-http.cpp)
+	# Use set_target_properties instead of target_compile_options to make CMake <= 2.8.11 happy
+	set_target_properties(test-asebahttp PROPERTIES COMPILE_FLAGS -std=c++11)
```

Comment:
I ran into philsquared/Catch#424 when building on Fedora 22. Note that the problem isn't with my particular machine as is suggested in that issue. I saw the problem on the build farm too. I suspect it is a GCC 5 bug.